### ISSUE TITLE
applying configuration failed

### ISSUE THREAD

**Issue by Dieterbe** [raintank/raintank-docker#25]
just saw this during `./build_all.sh`

```
Applying plugin configuration to rabbit@7b090dbfcf59... failed.
 * Could not contact node rabbit@7b090dbfcf59.
   Changes will take effect at broker restart.
 * Options: --online  - fail if broker cannot be contacted.
            --offline - do not try to contact broker.
```

is this harmless?

**Comment by woodsaj** [comment]
Yes this is expected and harmless. During the build we run rabbitmqctl to enable the mgmt plugin.  This will enable the plugin in the config and also try to contact the running rabbit (which doesnt exist) to notify it of the change.

When the image is started the mgmt plugin will be enabled because it is enabled in the config.

**Comment by woodsaj** [comment]
Actually we should just pass the `--offline` option so rabbitmqctl doesnt try and contact the broker (rabbit server)
